code/Player/Cameras/SpectatorCamera.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using Sandbox;
+
+namespace ssl.Player.Cameras
+{
+    public partial class SpectatorCamera : Camera
+    {
+        private const float SpectatorFieldOfView = 80F;
+        private const float FocusDistance = 100F;
+        private const float MaxSpeed = 75F;
+        private const float SpeedChangeFactor = 3F;
+
+        private bool clearTargetPressed;
```

Comment:
isClear..

Reply:
This is to check if the button to clear the target is being pressed